src/components/settings/members/InviteItem.tsx

```diff
@@ -0,0 +1,124 @@
+import { forwardRef, MutableRefObject } from 'react'
+import styled from 'styled-components'
+import { gql } from '@apollo/client'
+
+import { theme } from '~/styles'
+import { Avatar, Typography, Skeleton, Button, Tooltip, Chip } from '~/components/designSystem'
+import { InviteItemFragment } from '~/generated/graphql'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { addToast } from '~/core/apolloClient'
+
+import { RevokeInviteDialogRef } from './RevokeInviteDialog'
+
+gql`
+  fragment InviteItem on Invite {
+    id
+    email
+    token
+    organization {
+      id
+      name
+    }
+  }
+`
+
+interface InviteItemProps {
+  invite: InviteItemFragment
+}
+
+export const InviteItem = forwardRef<RevokeInviteDialogRef, InviteItemProps>(
+  ({ invite }: InviteItemProps, ref) => {
+    const { id, email, token, organization } = invite
+    const { translate } = useInternationalization()
+
+    return (
+      <ItemContainer>
+        <LeftBlock>
+          <Avatar variant="user" identifier={email.charAt(0)} size="medium" />
+          <Typography variant="body">{email}</Typography>
+        </LeftBlock>
+        <RightBlock>
+          <Chip label={translate('text_63208b630aaf8df6bbfb2665')} />
+          <Tooltip placement="bottom-end" title={translate('text_63208b630aaf8df6bbfb265f')}>
+            <ActionButtonIcon
+              icon="duplicate"
+              variant="quaternary"
+              onClick={() => {
+                navigator.clipboard.writeText(`${window.location.origin}/invitation/${token}`)
```

Comment:
Ok so you're reusing it as well, I think it either should be in the `externalUrl` file or at least export it from one of the file to reuse it in the other.
Right now if the url changes somwhere it's gonna be annoying to make sure it's good everywhere and can lead to bugs

Reply:
Will update according to https://github.com/getlago/lago-front/pull/415#:~:text=ansmonjol-,now,-Will%20use%20generatePath